pkg/cache/context.go

```diff
@@ -472,25 +457,26 @@ func (ctx *Context) AssumePod(name string, node string) error {
 			}
 			// assign the node name for pod
 			assumedPod.Spec.NodeName = node
-			return ctx.schedulerCache.AssumePod(assumedPod, allBound)
+			ctx.schedulerCache.AssumePod(assumedPod, allBound)
+			return nil
 		}
 	}
 	return nil
 }
 
 // forget pod must be called when a pod is assumed to be running on a node,
 // but then for some reason it is failed to bind or released.
-func (ctx *Context) ForgetPod(name string) error {
+func (ctx *Context) ForgetPod(name string) {
 	ctx.lock.Lock()
 	defer ctx.lock.Unlock()
 
 	if pod, ok := ctx.schedulerCache.GetPod(name); ok {
 		log.Logger().Debug("forget pod", zap.String("pod", pod.Name))
-		return ctx.schedulerCache.ForgetPod(pod)
+		ctx.schedulerCache.ForgetPod(pod)
+		return
 	}
 	log.Logger().Debug("unable to forget pod",
 		zap.String("reason", fmt.Sprintf("pod %s not found in scheduler cache", name)))
```

Comment:
NIT: Can we clean up this logging to just be:
```
log.Logger().Debug("unable to forget pod: not found in cache", zap.String("pod", name))
```

Reply:
Updated.